app/api/coreApi.js

```diff
@@ -523,9 +523,7 @@ function getBlock(blockHash, full = false) {
 function getBlocks(blockHashes, full = false) {
 	return new Promise(function(resolve, reject) {
 		Promise.all(blockHashes.map(h => getBlock(h, full))).then(function(blocks) {
-			var result = {};
```

Comment:
Only one call site existed that expected this to return a map by block hash. I changed that call site to build the map itself.

Reply:
fair